tests/unit/test_airflow.py

```diff
@@ -448,3 +446,121 @@ def test_amazon_alg_training_config_all_args(sagemaker_session):
     }
 
     assert config == expected_config
+
+
+def test_framework_tuning_config(sagemaker_session):
+    mxnet_estimator = mxnet.MXNet(
+        entry_point="{{ entry_point }}",
+        source_dir="{{ source_dir }}",
+        py_version='py2',
```

Comment:
I know it's just a unit test, but it does seem a bit strange to me we're still adding new code that "uses" Python 2

Reply:
Updated.